File: app/src/main/java/io/github/ketzalv/validationedittext/sample/utils/AlertUtils.java
package io.github.ketzalv.validationedittext.sample.utils;

import android.content.Context;
import android.content.DialogInterface;

import androidx.annotation.StringRes;
import androidx.appcompat.app.AlertDialog;

import io.github.ketzalv.validationedittext.sample.R;

public class AlertUtils {

    public static void showGenericAlert(Context context, boolean cancelable, @StringRes int title, String message) {
        AlertDialog.Builder builder = new AlertDialog.Builder(context, R.style.Theme_MaterialComponents_Light_Dialog_Alert);
        builder.setTitle(title)
                .setCancelable(cancelable)
                .setMessage(message)
                .setPositiveButton(R.string.accept, new DialogInterface.OnClickListener() {
                    @Override
                    public void onClick(DialogInterface dialog, int which) {

                    }
                })
                .setOnCancelListener(new DialogInterface.OnCancelListener() {
                    @Override
                    public void onCancel(DialogInterface dialog) {

                    }
                })
                .create()
                .show();
    }
}

File: sample/src/main/java/io/github/ketzalv/validationedittext/sample/utils/AlertUtils.kt
package io.github.ketzalv.validationedittext.sample.utils

import android.content.Context
import androidx.annotation.StringRes
import androidx.appcompat.app.AlertDialog
import io.github.ketzalv.validationedittext.sample.R

object AlertUtils {
    fun showGenericAlert(context: Context?, cancelable: Boolean, @StringRes title: Int, message: String?) {
        val builder = AlertDialog.Builder(context!!, R.style.Theme_MaterialComponents_Light_Dialog_Alert)
        builder.setTitle(title)
                .setCancelable(cancelable)
                .setMessage(message)
                .setPositiveButton(R.string.accept) { dialog, which -> }
                .setOnCancelListener { }
                .create()
                .show()
    }
}